docs/content/en/docs/yaml-crd-ref/metricsprovider.md

```diff
@@ -30,6 +30,7 @@ spec:
   secretKeyRef:
     name: <secret-name>
     key: <secret-key-that-holds-token>
+    optional: true
```

Comment:
```suggestion
    optional: true | false
```
This is the synopsis so you need to show all allowable values

Reply:
Ok but how can i show previous table in this